### DISCUSSION TITLE
Will there be any updates for this wonderful script?

### DISCUSSION THREAD

**Discussion by Pochemy-ux** [pierrelasse/YimStuff#22] [Q&A]
Will there be any updates for this wonderful script?

**Comment by pierrelasse** [comment]
Hi, thanks for the nice words.
There are multiple reasons I stopped developing the script.

TL;TR: gta boring for me 😦 so no motivation 😢 

One of them is that I got bored of GTA V (I have 1186h playtime). The game itself has nothing big to offer and the only entertainment were the interactions with players (helping them in general or by playing missions (cayo) to earn them money, trolling & streamsniping). Another reason is that the YimMenu repo got termed, I know there is a new one from Mr-X-GTA but it only has him as a developer and a way smaller community. When I tried using his version a few days ago, It didn't work as the game version was too high. The reason the YimMenu/YimMenu repo got termed is due to Cockstar arguing that they used files from the gta source code leak. So why couldn't they remove these "classes", "reverse" them themself and get it the repo back up. But whatever. Many other ppl I know don't play GTA anymore due to Blindeye. I don't even know if any menu works atm or what the situation is. What I tried to do in the script is to combine the best features from all other scripts and menus and whatever (even fivem scripts), so that you would only have to use this one. While also keeping everything the same style, and making everything as high quality as possible. Another thing - that probably doesn't have to do with me being inactive, is that the only feedback is from the issues, discussions, and sometimes recommending it somewhere (found [this](https://www.reddit.com/r/YimMenu/comments/1efmbjd/any_more_addons/) 1mo ago lol). And the cursed "insights" from GitHub.
*open the image in a new tab for it to be bigger*
<img src="https://github.com/user-attachments/assets/7ca48e6c-c124-4ca3-87e9-f6292e7eae13" width="330px">
*YimMenu installation tutorial more famous than the script installation tutorial💀*
Close to every time the game updated with yet another "low effort" / not very innovative business, you would have to update the globals, locals, etc. Which I mostly permanently borrowed from other scripts 🙂. The Thorben Discord got "sold" or whatever, so I had even less contact with GTA ppl. Atm (no one prob cares about this) I watch YouTube, sometimes play Minecraft even tho it gotted really boring too, as there are no really good servers to play on. Minehut; dumpster fire where most of them are 3rd party hosted and are all p2w and on minehut to get players. And I don't know of any good multiplayer games that make me fun. But yh. Enough with slapping you with big text brick.

**Reply by Pochemy-ux** [reply]
This is of course sad because I really wanted this script to be updated because it has a lot of functions, half of which are not in other scripts and in the menu itself, and I have been using it since the moment when I almost quit the game because there was nothing to do in it, but then I discovered the world of cheats and played about a thousand more hours. I use the Chronix menu, this is the same YIM menu, only it is updated more often, judging by their GitHub, and I thought maybe you also switched to this menu and would give a link to the new version of this script, but alas, it was not meant to be)

**Reply by pierrelasse** [reply]
Where do I get the menu?

**Reply by KLXYinc** [reply]
womp womp

**Reply by pierrelasse** [reply]
![tenor](https://github.com/user-attachments/assets/491e5815-7da0-477c-8031-a03e01aef28e)

**Reply by Pochemy-ux** [reply]
> Где взять меню?

https://github.com/Deadlineem/Chronix
